realm/realm-library/src/main/java/io/realm/log/RealmLog.java

```diff
@@ -51,6 +51,9 @@ public static void add(RealmLogger logger) {
      * @param level see {@link LogLevel}.
      */
     public static void setLevel(int level) {
+        if (level < LogLevel.ALL|| level > LogLevel.OFF) {
```

Comment:
Only saw it once, so no, but in any case, it is a nice check to have since the method is public and the input parameter is `int`